release/test_release_gold.sh

```diff
@@ -7,35 +7,39 @@ source release_gold.sh --test
 function main {
 	set_up
 
-	if [ "${?}" -ne 0 ]
+	test_not_reference_new_releases
+	test_reference_new_releases
+
+	if [ -d "${_PROJECTS_DIR}/liferay-portal-ee" ]
 	then
-		return
+		test_not_prepare_next_release_branch
+		test_not_update_release_info_date
+		test_prepare_next_release_branch
+		test_update_release_info_date
+	else
+		echo -e "The directory ${_PROJECTS_DIR}/liferay-portal-ee does not exist.\n"
 	fi
 
-	test_not_prepare_next_release_branch
-	test_not_update_release_info_date
-	test_prepare_next_release_branch
-	test_update_release_info_date
-
 	tear_down
 }
 
 function set_up {
+	export LIFERAY_RELEASE_PRODUCT_NAME="dxp"
+	export LIFERAY_RELEASE_RC_BUILD_TIMESTAMP="1695892964"
 	export _PROJECTS_DIR="${PWD}"/../..
-
-	if [ ! -d "${_PROJECTS_DIR}/liferay-portal-ee" ]
-	then
-		echo -e "The directory ${_PROJECTS_DIR}/liferay-portal-ee does not exist.\n"
-
-		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
-	fi
 }
 
 function tear_down {
-	lc_cd "${_PROJECTS_DIR}/liferay-portal-ee"
+	lc_cd "${_PROJECTS_DIR}/liferay-docker"
+
+	git restore .
+
+	lc_cd "${_PROJECTS_DIR}/liferay-portal-ee" 2>/dev/null
```

Comment:
@kiwm, did you have the chance to see this [comment](https://github.com/liferay-release/liferay-docker/pull/69#discussion_r1885811290)?